admin-manual/installation-setup/customization/scaling-archivematica.rst

```diff
@@ -0,0 +1,396 @@
+.. _scaling-archivematica:
+
+=====================
+Scaling Archivematica
+=====================
+
+.. _scaling-overview:
+
+Overview
+--------
+
+Archivematica includes a number of features and options to improve scalability
+& performance. The default configuration options are geared towards “typical” 
+use cases and for many users won’t require any modification. However, some 
+users will benefit from optimizing configuration options to meet their 
+specific preservation needs & available computing resources.
+
+This guide describes three broad approaches to scaling & optimization and the 
+specific techniques available to each approach. 
+
+This is not a detailed ‘how to’ guide or set of specific recommendations. The 
```

Comment:
Same here! You may want to do a quick pass through the document to make sure we fix this!

Reply:
yep, there will be at least a few examples of this... I'll go through and fix them all at once